rand_xoshiro/src/common.rs

```diff
@@ -224,6 +224,14 @@ macro_rules! impl_xoshiro_large {
 
 /// Map an all-zero seed to a different one.
 macro_rules! deal_with_zero_seed {
+    ($seed:expr, $Self:ident, $bytes:expr) => {
+        if $seed == [0; $bytes] {
+            return $Self::seed_from_u64(0);
+        }
+    }
+}
+
+macro_rules! deal_with_zero_seed_large {
```

Comment:
These could be the same macro just by deleting lines `232..=234`.

Reply:
Ooooooh, didn't know you could do that!